src/borg/archiver.py

```diff
@@ -2733,6 +2733,8 @@ def build_parser(self):
             'a_status_oddity': '"I am seeing ‘A’ (added) status for a unchanged file!?"',
             'separate_compaction': '"Separate compaction"',
             'list_item_flags': '"Item flags"',
+            'key_files': 'Internals -> Data structures and file formats -> Key files',
+            'borg_key_export': 'borg key export --help'
```

Comment:
if one adds a trailing comma after the last element, future changes are easier / diffs are prettier.

Reply:
see your own change above: you did not need to edit line 2735, because the comma was already there.

Reply:
Done.